lib/experimental/Widgets/Layout/Dashboard/index.tsx

```diff
@@ -39,7 +39,7 @@ const DashboardComponent = forwardRef<HTMLDivElement, DashboardProps>(
     }, [setColumns])
 
     return (
-      <div ref={ref} className="overflow-hidden">
```

Comment:
This is needed to prevent shadows to be cropped on the sides

Reply:
Unfortunately this also means that we are dropping the scroll bars 👀

Reply:
can we add some padding to accommodate the shadows?

Reply:
But then we would need to add negative margin to the container. I think the best solution, as spoken with @sauldom102 is to make the parent (container) dictate the shadows via `[&>*]:shadow-md`. The dashboard benefits from shadows, but a strip doesn't.

If you think about it, it makes sense that this is the responsability of the parent, the same way gap or overall layout is.

Reply:
Sorry, I'm mixing things:

1. Yes we need to remove `overflow-hidden` from dashboard, 100%
2. We can't have widgets have shadows, we need to let the container decide.